resources/application-connector/values.yaml

```diff
@@ -39,16 +39,16 @@ global:
     version: e029fcf4
```

Comment:
How did this file end up in this PR?

Reply:
we need to force the image update in application operator upgrade test.

Reply:
It is done to fix Application Operator upgrade tests, as it is not passing.